specs/LocaleRegion.md

```diff
@@ -0,0 +1,111 @@
+LocaleRegion
+===
+
+# Background
+Developers have requested access to programmatically choose the specific locale/region to use in WebView2. 
+The locale/region updates the JavaScript region (objects in the ECMA standard) which gets reflected as
+variations in time and date formats. Currently, the locale is by default set to the same value as the 
+display language. 
+
+You can use the `LocaleRegion` API to update the locale value individually from the display
+language.
+
+# Description
+* The intended value for `LocaleRegion` is in the format of `language[-country]` where `language` is the 
+2-letter code from [ISO 639](https://www.iso.org/iso-639-language-codes.html) and `country` is the 
+2-letter code from [ISO 3166](https://www.iso.org/standard/72482.html).
+
+# Examples
+```cpp
+auto webViewEnvironment10 = m_webViewEnvironment.try_query<ICoreWebView2Environment10>();
+wil::com_ptr<ICoreWebView2ControllerOptions> options;
+HRESULT hr = webViewEnvironment10->CreateCoreWebView2ControllerOptions(&options);
+options->put_LocaleRegion(m_webviewOption.localeRegion.c_str());
+```
+
+```c#
+CoreWebView2Environment environment;
+CoreWebView2ControllerOptions CreateCoreWebView2ControllerOptions(CoreWebView2Environment environment)
+{
+    CoreWebView2ControllerOptions ControllerOptions = null;
+    if (LocaleRegion != null)
+    {
+        ControllerOptions = environment.CreateCoreWebView2ControllerOptions();
+        ControllerOptions.LocaleRegion = LocaleRegion;
+    }
+    return ControllerOptions;    
+}
+```
+
+# API Details
+```cpp
+[uuid(0c9a374f-20c3-4e3c-a640-67b78a7e0a48), object, pointer_default(unique)]
+interface ICoreWebView2StagingControllerOptions : IUnknown {
+  /// The default region for the WebView2.  It applies to JavaScript API
+  /// Intl.DateTimeFormat() which affects string formatting like

```

Comment:
It applies to everything under Intl correct? It would be nice to adjust this to say just that it sets the default locale for all Intl JavaScript APIs and other JavaScript APIs that depend on that like Date.toLocaleDateString. And provide a short one line example of JavaScript that would be affected like `Intl.DateTimeFormat().format(new Date())` (or appropriate)